### DISCUSSION TITLE
PHP constant for Project Config path?

### DISCUSSION THREAD

**Discussion by lindseydiloreto** [craftcms/cms#9036] [Ideas]
### Description

As a tangential follow-up to #7921...

I see that there is a `CRAFT_CONFIG_PATH` constant to specify which `config` folder we want to use. However, I only want to move the `config/project` folder... I do not want to reference anything else in this particular `config` folder.

Would it be possible to have a constant that specifically alters the path of the `config/project` directory?

**Comment by brandonkelly** [comment]
The `project/` folder name is already configurable via `craft\services\ProjectConfig::$folderName`, but it’s assumed that the folder will live within the `config/` folder. So changing that would be a potentially breaking change. Earliest we could make the full path to the folder configurable would be v4.

**Comment by lindseydiloreto** [comment]
Interesting... I can find a workaround until then. Thanks!
